app/src/main/java/org/akvo/flow/ui/view/CascadeQuestionView.java

```diff
@@ -254,7 +254,8 @@ public void resetQuestion(boolean fireEvent) {
         super.resetQuestion(fireEvent);
         updateSpinners(POSITION_NONE);
         if (mDatabase == null) {
-            String error = "Cannot load cascade resource: " + getQuestion().getSrc();
+            String error = getContext()
```

Comment:
This is a bit out of scope but I ectracted a hardcoded string so it can also be translated.